src/transport_picoquic.cpp

```diff
@@ -620,7 +621,7 @@ PicoQuicTransport::enqueue(const TransportConnId& conn_id,
                       std::move(bytes),
                       std::move(trace)};
 
-        data_ctx_it->second.tx_data->push(std::move(cd), ttl_ms, priority);
+        data_ctx_it->second.tx_data->push(std::move(cd), ttl_ms, priority, delay_ms);
 
```

Comment:
Do we want to make it 0 as well

Reply:
thanks for catching that. It should have been zero for this one.